esbuild.server.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const dotenv = require('dotenv');
+
+// Pulls in .env and populates the define object.
+// This is used to populate environment variables at built time.
+dotenv.config();
+const define = {};
+for (const k in process.env) {
+  // eslint-disable-next-line functional/immutable-data
+  define[`process.env.${k}`] = JSON.stringify(process.env[k]);
+}
+
+const watchMode = process.env['WATCH'];
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('esbuild').build({
+  entryPoints: ['./src/server/index.ts'],
+  bundle: true,
+  minify: true,
+  platform: 'node',
+  external: ['fsevents', 'esbuild'],
+  outfile: 'build/server.js',
+  target: 'node10.4',
```

Comment:
This should be node 14 lts, as that's what we're running gateway on

Reply:
Is there a dept. wide supported version of Node? It might be `14.16.1`?

It's ever so slightly annoying having to `nvm use` when switching projects - not a big deal though

Reply:
good catch

Reply:
@oliverlloyd it is frustrating, there is a shell script that nvm provide which looks for an `nvmrc` and auto-`nvm use`s but I've found it can slow down my terminal a little

Reply:
honestly hard coding it is probably fine

Reply:
there isn't really a standard version, the general advice is to use the latest LTS  version if possible

Reply:
I'll go ahead and set the target to `node14` if we're all in agreement? :)

Reply:
Or maybe we can use 14.16.1 just so we're aligned with DCR and friends?

Reply:
good suggestion, pushed that up